### DISCUSSION TITLE
the "Theme Chooser" button does not appear on my account

### DISCUSSION THREAD

**Discussion by danielbrown1231** [community/community#33650] [New to GitHub]
I created a Github with the hopes of hosting a static site using github pages and I want to use a jekyll theme, but the theme chooser button is not visible on my github pages.
![Screenshot 2022-09-20 203032](https://user-images.githubusercontent.com/114035268/191394183-0b4a7beb-9fbb-48da-937b-6046517a1594.png)

**Comment by radar-blip** [comment]
You're right, it looks like they did away with that feature. 
Looks like you can still do it though, if you follow the directions under the 'add a Jekyll theme' link.
I haven't done it myself but it shouldn't be to difficult.
